styles.css

```diff
@@ -1,15 +1,1236 @@
-body {
-  box-sizing: border-box;
-  overflow-x: hidden;
-  background-color: black;
+@import url(https://fonts.googleapis.com/css?family=Roboto:400,300,500);
+ *{
+     margin: 0;
+     padding: 0;
+     box-sizing: border-box;
+     list-style-type: none;
+ }
+ 
+ body{
+     
+    height: 100vh;
+    width: 100%;
+    background-color: #374859;
+    font-family: 'Quicksand', sans-serif;
+    
+ }
+   
+ header{
+     display: flex;
+     flex-direction: column;
+     width:100%;
+     height:17vh;
+ }
+ .nav{
+     display: flex;
+     height:8vh;
+     background-color:  white
+ }
+ .second-row{
+     background-color:  #f4f4f4f4;
+     display: flex;
+     flex-direction:row;
+ }
+ .second-roww{
+    background-color:  #f4f4f4f4;
+    display: flex;
+     flex-direction:row;
+}
+ .logo{
+     display: flex;
+     flex:2;
+     padding: 10px 30px;
+     align-items: center;
+ }
+ #logo{
+     color: blue;
+     font-family: 'Pattaya', sans-serif;
+     text-shadow: 
+        2px 2px 0 #18e657,
+        4px 4px 0 #bf189b,
+        6px 6px #ffeb00,
+        8px 8px 0 black
+ }
+ .drop-down{
+     display:flex;
+     flex:5;
+     justify-content: space-between;
+     align-items: center;
+     margin-right: 13%
+ }
+  
+ .socials-ul{
+     flex: 1;
+    display: flex;
+    flex-direction: row;
+    margin-right: 20px;
+    padding-left: 1%;
+    justify-content: space-between;
+    align-items: center;
+ }
+ #twitter{
+    color: 	#1DA1F2
+}
+ #insta{
+    background: -webkit-linear-gradient(#F58529,#FEDA77,#DD2A7B,#8134AF,#515BD4, red);
+   -webkit-background-clip: text;
+   -webkit-text-fill-color: transparent;
+   border-radius: 5px
+}
+#link{
+   color: #0e76a8
+}
+#discord{
+   color: #7289d9
+}
+ .toggle{
+     display: none
+ }
+a{
+    text-decoration: none;
+    color: #ccc
+}
+a:hover{
+    text-decoration: none;
+}
+a:active{
+    text-decoration: none;
+}
+a:visited{
+    text-decoration: none;
+}
+/* seacrh and cart */
+.second-row{
+    display: flex
+}
+.ccc{
+  cursor: pointer;
+}
+.searchh{
```

Comment:
why extra h and letters on these classes?

Reply:
this is because earlier on i considered using bootstrap which messed up with existing class  names like card and search so i added an extra last letter